src/main/java/io/jenkins/plugins/opentelemetry/backend/grafana/LokiBuildLogsLineIterator.java

```diff
@@ -24,7 +24,6 @@
 import java.util.Optional;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import javax.annotation.Nonnull;
```

Comment:
Many of these classes already had the SpotBugs imports and were using them in addition to the JSR-305 imports within the same class.